docs/src/main/asciidoc/logging.adoc

```diff
@@ -418,6 +414,7 @@ For details about its configuration, see the xref:#quarkus-log-logging-log-confi
 
 By default, the file log handler in Quarkus is disabled.
 Once enabled, it enables the logging of all events to a file on the application's host, while also supporting log file rotation.
+
```

Comment:
The previous sentence ends with mentioning log file rotation, so why do we start a new paragraph here?

We could alternatively move the first mention of log rotation to the next paragraph.

Reply:
Abstracts are limited to 26 words. Empty lines improve readability by dividing paragraphs into smaller chunks suitable for the reference style.

However, level 3 headings (=== XXX) don't require an abstract since they are subsections of a chapter with an abstract, so here it was just about that splitting.

What I will do is to patch this section so that it begins in more literal way :) Thank you for pointing this out.

Reply:
My applied tweak:

```
The Quarkus file log handler is disabled by default, but when enabled, it logs events to a file on the host, supporting log file rotation.

Log file rotation ensures effective log file management over time by maintaining a specified number of backup log files, while keeping the primary log file up-to-date and manageable.
```
@Ladicek If OK in your books, please, close the thread. :)

Reply:
I think the previous version was better. Since we're not creating an abstract here, we don't need to condense as much information as possible into a single sentence.

Here's what I'd use:

> By default, the file log handler in Quarkus is disabled.
> Once enabled, it writes log messages to a file on the application's host.
>
> The file log handler also supports log file rotation.
> Log file rotation ensures effective log file management over time by maintaining a specified number of backup log files, while keeping the primary log file up-to-date and manageable.